src/locales/en.json

```diff
@@ -76,12 +76,13 @@
     "primary_biometry_android": "Unlock with my biometric fingerprints",
     "primary_biometry_faceid": "Unlock with my Face ID",
     "primary_biometry_touchid": "Unlock with my Touch ID",
+    "primary_biometry_denied_title": "Unlock with my biometric fingerprints",
```

Comment:
Is it still needed?

Reply:
Good catch, was from a first attempt before choosing to create the `BiometryDeniedDialog` section. Fixed